### PR TITLE
Adds support for different Forms types

### PR DESCRIPTION
**Pull request by david1asher**
Asana: https://app.asana.com/0/280843276935180/797693123667337/f

Typeform is a platform that helps sending forms for users/people to fill up,
sometimes they're not filling up the form, so in order to specify which forms to fetch we need to add some query-params to our api call, t
hats what Ive added here, choosing which forms to fetch will be specified via this flag: `__formTypes`
In addition added the flag to the doc containing all the flags here : https://docs.google.com/spreadsheets/d/1VXoSCLMNeB9I_OELKj9iIM3lzA1RQyxU9eEA4zGNuWM/edit#gid=0

### CODE REVIEW

File: typeform/typeform.py [hunk 1]
```diff
@@ -113,12 +120,15 @@ def _request(self, url, params=None):
     def _build_params(self, form, batch_size):
         """ construct the relevant params according to the Typeform API """
         page_size = batch_size if batch_size else BATCH_SIZE
+        completed = FORM_TYPES[self.source.get('__formTypes')]
         params = {
             'page_size': page_size,
             'sort': 'landed_at,desc',
-            'completed': 1,  # remove the line if you want to include both
+            # 'completed': 1,  # remove the line if you want to include both
```
Comment (GregRosPanoply) [inline typeform/typeform.py:127]: No reason to keep this line.

**Review by GregRosPanoply — APPROVED** [review]
Looks good! Just a small comment.

File: typeform/typeform.py (hunk 1 above)
Comment (german-panoply) [inline typeform/typeform.py:127]:
```suggestion
```

**Review by german-panoply — APPROVED** [review]

File: typeform/typeform.py (hunk 1 above)
Comment (german-panoply) [reply typeform/typeform.py:127]: I'll change it in the next PR since I don't permissions to commit to David's branch.

**Pull request merged**